arviz/plots/dotplot.py

```diff
@@ -0,0 +1,229 @@
+import numpy as np
+
+from ..rcparams import rcParams
+from .plot_utils import get_plotting_function
+
+
+def plot_dot(
+    values=None,
+    binwidth=None,
+    dotsize=1,
+    stackratio=1,
+    hdi_prob=None,
+    rotated=False,
+    dotcolor="C0",
+    intervalcolor="C3",
+    markersize=None,
+    markercolor="C0",
+    marker="o",
+    figsize=None,
+    linewidth=None,
+    point_estimate="auto",
+    nquantiles=50,
+    quartiles=True,
+    point_interval=None,
+    ax=None,
+    show=None,
+    plot_kwargs=None,
+    backend=None,
+    backend_kwargs=None,
+    **kwargs
+):
+
+    """Plot distribution as dot plot or quantile dot plot.
+
+    This function uses the Wilkinson's Algorithm
+    (Leland Wilkinson (1999) Dot Plots, The American Statistician, 53:3, 276-281,
+    DOI: 10.1080/00031305.1999.10474474)to allot dots to bins.
```

Comment:
How about using citation like references? 

```suggestion
    This function uses the Wilkinson's Algorithm [1]_ to allot dots to bins.
```

plus the complete reference in the `References` section like in https://arviz-devs.github.io/arviz/_modules/arviz/plots/separationplot.html#plot_separation. Same for the paper below, maybe leaving the title and reference number in the text?